ctam/interfaces/functional_ifc.py

```diff
@@ -631,16 +630,23 @@ def RedfishDownloadDump(self, DumpURI):
             print(str(e))
             return None
     
-    def ctam_monitor_task(self, TaskID):
+    def ctam_monitor_task(self, TaskID=""):
         """
         :Description:       CTAM Monitor a Task
         :param TaskID       Task ID of TaskService/Tasks (for accelerator management) to monitor
         :returns:	        (Task_Completed, JSONData response)
         :rtype:             Tuple
         """
         Task_Completed = False
+        JSONData = {}
+        if not TaskID:
+            # need to implement the flow for no task uri
+            self.test_run().add_log(LogSeverity.DEBUG,
+                    "No Task ID provided."
+                )
+            return Task_Completed, JSONData
         TaskURI = self.dut().uri_builder.format_uri(
-            redfish_str="{BaseURI}/TaskService/Tasks/" + "{}".format(TaskID), component_type="GPU"
+            redfish_str="{BaseURI}{TaskServiceURI}" + "{}".format(TaskID), component_type="GPU"
```

Comment:
@v-ajitpanda Can you please add TaskServiceURI in json spec and test your changes thoroughly? Thanks.

Reply:
Okay! Can you test the binary once for all of your changes.?

Reply:
We are still having trouble with binary creation in windows. We can resolve this for the next commit.